graph_nets/demos_tf2/shortest_path.ipynb

```diff
@@ -367,33 +367,64 @@
     "    graphs.append(graph)\n",
     "  return input_graphs, target_graphs, graphs\n",
     "\n",
-    "\n",
-    "def create_placeholders(rand, batch_size, num_nodes_min_max, theta):\n",
-    "  \"\"\"Creates placeholders for the model training and evaluation.\n",
+    "def networkxs_to_tf_graphs_tuple(graph_nxs,\n",
+    "                              node_shape_hint=None,\n",
```

Comment:
Make sure arguments are aligned, and leave exactly two empty lines between method definitions :)